src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -297,19 +297,21 @@ enum endless_action_type {
 #define FILE_GZ_EXTENSION	"gz"
 #define FILE_XZ_EXTENSION	"xz"
 
-#define RELEASE_JSON_URLPATH    _T("https://d1anzknqnc1kmb.cloudfront.net/")
-#define JSON_LIVE_FILE          "releases-eos-3.json"
-#define JSON_INSTALLER_FILE     "releases-eosinstaller-3.json"
-#define JSON_GZIP               "." FILE_GZ_EXTENSION
-#define JSON_PACKED(__file__)   __file__ JSON_GZIP
+#define RELEASE_JSON_URLPATH       _T("https://d1anzknqnc1kmb.cloudfront.net/")
+#define JSON_LIVE_FILE             "releases-eos-3.json"
+#define JSON_CODING_LIVE_FILE      "releases-coding-3.json"
+#define JSON_INSTALLER_FILE        "releases-eosinstaller-3.json"
+#define JSON_CODING_INSTALLER_FILE "releases-codinginstaller-3.json"
```

Comment:
This is a very insightful clarification, thanks! I'll remove the `codinginstaller`-fu.